kotkt.nl/content/_index.en.md

```diff
@@ -104,7 +104,7 @@ The association tries to give every running campaign a space in Luna where the c
 
 At the Knights there are also fans of Life Action Roleplaying, more commonly known as LARP. A Larp is similar to tabletop roleplaying, where instead one describes to do actions, one actually uses the entire body to simulate the event happening.
 
-Twice-yearly we host our own one-shot LARPs which take place during our own association weekends. Every member gets a unique character in the world, varying from fairytale figures during a quarantine to survivors of a destroyed cruiseship. Furthermore our association aids in making and expanding LARP costumes. Every 1st Sunday of the month the Brass Scissors order is present where members can go to for aid in where one can buy the best material, as well as working and sowing the materials together to form your LARP costume.
+Our association aids in making and expanding LARP costumes. Every 1st Sunday of the month the Brass Scissors order is present where members can go to for aid in where one can buy the best material, as well as working and sowing the materials together to form your LARP costume.
```

Comment:
It's unclear for externals who/what is the Brass Scissors, this may be read as an external organisation.
```suggestion
Our association aids in making and expanding LARP costumes. Every 1st Sunday of the month members gather for the Brass Scissors day where members can go to for aid in where one can buy the best material, as well as working and sowing the materials together to form your LARP costume.
```